resources/config-defaults.edn

```diff
@@ -292,6 +292,9 @@
  ;; Show "Assign external id" action button in UI for handlers
  :enable-assign-external-id-ui false
 
+ ;; maximum size of attachment files in bytes (10485760 = 10MB)
+ :attachment-max-size 10485760
```

Comment:
this will introduce a breaking change since now uploads have basically no size limit. 10MB is a size i have seen used in many web servers and that is why it's in here, but it could also be a good discussion if this should be an optional value

Reply:
We can put it in the release notes very visibly at least. @jaakkocsc you can decide if this is a problem or not. We can leave the size undefined / infinite but I think this is large enough not to cause problems. We could talk to the actual instance owners about this.

Reply:
We could at least make sure that `nil` is documented here (as infinite) and that it works (test it).

Reply:
Should we use 10 * 1000 * 1000 here?

Reply:
And we can add the comment `nil` means not limited.